modules/abstract-utxo/src/recovery/backupKeyRecovery.ts

```diff
@@ -62,16 +55,17 @@ export interface FormattedOfflineVaultTxInfo {
  * @param txHex
  * @returns {{txHex: *, txInfo: {unspents: *}, feeInfo: {}, coin: void}}
  */
-function formatForOfflineVault<TNumber extends number | bigint = number>(
```

Comment:
why would you take away `TNumber` here?

Reply:
As per new change, `OfflineVaultTxInfo` uses valueString to carry large values. value is always `number`. So no need to for Generics.